### ISSUE TITLE
Can you share the pre-trained model in coco2017?

### ISSUE THREAD

**Issue by YexiongLin** [Okery/PyTorch-Simple-MaskRCNN#7]
Can you share the pre-trained model in coco2017?

**Comment by WWaynee** [comment]
Here you are>>
link: https://pan.baidu.com/s/1CfAFj30PdtP5zXRlCgPnUA
access code: 97f9 
And I have trouble training the model, Can I communicate with you on this project? I received a task and need to learn Mask R-CNN in object detection. My QQ number 1242654256.

**Comment by Okery** [comment]
Thank you, WWaynee.
If you have any trouble with this repo, you can add my QQ: 396968482
